zc_install/includes/functions/general.php

```diff
@@ -199,8 +199,13 @@ function getDetectedURIs($adminDir = 'admin') {
     $adminServer .= $httpServer;
     $catalogHttpServer = 'http://' . $httpServer;
     $catalogHttpUrl = 'http://' . $httpServer . '/' . zen_parse_url($url, 'path', true);
-    $catalogHttpsServer = 'https://' . $httpServer;
-    $catalogHttpsUrl = 'https://' . $httpServer . '/' . zen_parse_url($url, 'path', true);
+    if ($request_type == 'SSL') { 
+      $catalogHttpsServer = 'https://' . $httpServer;
+      $catalogHttpsUrl = 'https://' . $httpServer . '/' . zen_parse_url($url, 'path', true);
+    } else {
+      $catalogHttpsServer = 'http://' . $httpServer;
+      $catalogHttpsUrl = 'http://' . $httpServer . '/' . zen_parse_url($url, 'path', true);
+    }
```

Comment:
In the `else` can we just set them to the non-https variable?  This helps it be slightly clearer about the intent (think semantic), and eliminates a bit of duplication.

ie: 
```php
} else {
  $https = $http;
  $httpsUrl = $httpUrl;
}
```